lib/PermissionsTransformer.ts

```diff
@@ -4,8 +4,9 @@ const PermissionsTransformer = (prefix: string): ClassDecorator => {
     return (target: any) => {
         eta.object.recursiveKeys(target).forEach(keys => {
             const item = keys.slice(0, -1).reduce((p, v) => p[v], target);
-            if (typeof(item[keys[keys.length - 1]]) !== "string") return;
-            item[keys[keys.length - 1]] = prefix + keys.join("/");
+            const last_key = keys[keys.length - 1];
```

Comment:
Gonna be a nag again - I want to keep consistency across the codebase. We don't use snake-case anywhere (except in the database column names, but that's just because PostgreSQL requires escaping for upper-case letters).